Automattic/wordpress-activitypub
# Add a reaction block

This will add a block to display Likes and Reposts in a single block. 

Fixes #1057 

## Proposed changes:
* Adds a Reactions block
* It automatically gets hooked to be placed after the post content block
* Enable reactions by default!

A (not quite up to date) video of it in action:

https://github.com/user-attachments/assets/e8d381b2-9627-489d-a528-b080b0ab498a

### Other information:


## Testing instructions:
With this PR active, view a post on the frontend that already has a number of Likes and Reposts. The reactions block should be inserted directly after the post content:

<img width="341" alt="Screenshot 2024-12-12 at 16 04 57" src="https://github.com/user-attachments/assets/71ac36cd-5445-45c6-a011-0e0e0700dbf4" />

The block is automatically added via a block hook. It should be opted in, and can be disabled in the Single Post template in the Site Editor: (also note the dummy content in the Editor)

<img width="946" alt="Screenshot 2024-12-12 at 16 06 31" src="https://github.com/user-attachments/assets/06452dd9-9710-42a6-ba57-62d9e8a1ab9b" />

Every row of icons, if there are too many to fit the width, should limit the number of avatars shown so that none are cut off.

<img width="340" alt="Screenshot 2024-12-12 at 16 15 35" src="https://github.com/user-attachments/assets/5a279fcd-bc00-40d2-bbcf-46bd992b25e9" />

Avatars and names are shown if you click on the **n reposts** (or likes) label to the right of the avatars:

<img width="624" alt="Screenshot 2024-12-12 at 16 17 04" src="https://github.com/user-attachments/assets/ae07a294-1cf7-4baa-b4fa-a57b1c0bc3ef" />

On a post with no likes or reposts, nothing will be shown, not even the title.

The title ("Fediverse reactions" by default) is editable.
